README.md

```diff
@@ -6,106 +6,235 @@
 [![Dependencies](https://david-dm.org/netlify/zip-it-and-ship-it/status.svg)](https://david-dm.org/netlify/zip-it-and-ship-it)
```

Comment:
https://david-dm.org/netlify/zip-it-and-ship-it/status.svg and  https://david-dm.org/netlify/zip-it-and-ship-it are broken

Reply:
Looks like https://david-dm.org/ is out of business. Removing those links.